language-server/modules/langserver-core/src/main/java/module-info.java

```diff
@@ -17,6 +17,7 @@
     exports org.ballerinalang.langserver.config;
     exports org.ballerinalang.langserver.telemetry;
     exports org.ballerinalang.langserver.util to io.ballerina.language.server.simulator;
+    exports org.ballerinalang.langserver.command;
```

Comment:
Why do we expose this? Core components are usually not exposed and only the commons components are exposed.

Reply:
In order to apply the text edits I needed `org.ballerinalang.langserver.command.CommandUtil`. 
Is there any other way?

Reply:
Yes, command utility is an internal implementation. You can use the following approach.
`context.getLanguageClient().applyEdit(workspaceEdit)`